lib/test_suite/analyzer_verification/assert_call.ex

```diff
@@ -0,0 +1,34 @@
+defmodule ElixirAnalyzer.TestSuite.AnalyzerVerification.AssertCall do
+  @dialyzer generated: true
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise Pacman Rules
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+
+  assert_call "finds call to local helper function" do
```

Comment:
Would it be possible to add something that asserts a function wasn't called anywhere? This would be crucial for writing analyzers for exercises like the concept exercise for recursion or `list-ops`, to verify that functions from `Enum` aren't used.

Reply:
Yea I have something in the underlying code to do that, not sure if I exposed it yet.

Reply:
I've exposed this as `assert_no_call` for now